protocol/src/protocol.fbs

```diff
@@ -256,3 +256,9 @@ table Alert {
     message: Bytes;
 }
 
+table Identify {
+  // network name
+  name: string;
```

Comment:
I don't think the `name: string` is a good identity for the network, its size is unlimited. What about `id: u64`.

And what about adding another field `version` used for forking.